.tool-versions

```diff
@@ -1,3 +1,3 @@
-elixir 1.9.2-otp-22
-erlang 22.0.7
+elixir 1.10.2
```

Comment:
We should still use the `1.10.2-otp-22` version if we are using Erlang 22

Reply:
@dkarter that version isn't listed in `asdf list-all elixir`

Reply:
Looks like erlang and elixir are meant to be installed separately.

Reply:
Really? that's weird... I see it on my end:
<img width="398" alt="image" src="https://user-images.githubusercontent.com/551858/82372337-70f4bc80-99e1-11ea-95e8-b119b2b78d5f.png">